### DISCUSSION TITLE
Find and Replace

### DISCUSSION THREAD

**Discussion by x61** [brookhong/Surfingkeys#1433] [General]
How do you use Find and Replace functionality? Is this even possible?

**Comment by userrand** [comment]
Are you referring to editing an input text field? Like the one, you used to post this question? If so then you could use:
`
Ctrl+i`

which is written in the help (found by using ?) as :

`<Ctrl-i>`:  Go to the edit box with the vim editor

Then you can use search and replace commands from vim.

For example, using `:s/Go/kl/` one can replace the Go to kl.
